### PR TITLE
Add ANE to constructor

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to dotnet/docs#26763.

### CODE REVIEW

File: xml/System.Windows.Forms/DataGridViewColumnStateChangedEventArgs.xml [hunk 1]
```diff
@@ -87,6 +85,8 @@
         <param name="stateChanged">One of the <see cref="T:System.Windows.Forms.DataGridViewElementStates" /> values.</param>
         <summary>Initializes a new instance of the <see cref="T:System.Windows.Forms.DataGridViewColumnStateChangedEventArgs" /> class.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="dataGridViewColumn" />is <see langword="null" />.</exception>
```
Comment (RussKie) [inline xml/System.Windows.Forms/DataGridViewColumnStateChangedEventArgs.xml:89]:
```suggestion
          <paramref name="dataGridViewColumn" /> is <see langword="null" />.</exception>
```

**Review by RussKie — COMMENTED** [review]
Should we mention the change affect 6.0RC1 onwards?

**Comment by gewarren** [comment]
@RussKie Could you approve this one so I can merge it?

**Review by RussKie — APPROVED** [review]

**Comment by RussKie** [comment]
Apologies, I totally missed it.

**Pull request merged**